<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configuration</class>
 <widget class="QWidget" name="Configuration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="PB_Enregistrer">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Enregistrer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Annuler">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_Chemin">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>C:/RiotGames</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Parcourir">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Parcourir...</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RD_RiotGames">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Chemin &amp;RiotGames</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RD_CheminDirect">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>331</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Chemin Direct &amp;vers les images</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
